pkgs/tools/security/aflplusplus/default.nix

```
@@ -123,9 +123,9 @@ let
     };
 
     meta = {
-      description = ''
-        AFL++ is a heavily enhanced version of AFL, incorporating many features and
-        improvements from the community.
+      description = "
```

```suggestion
      description = ''
```

Multi-line strings require two single quotes.

---

Oops! Editor must have messed up quotes 😅